### PR TITLE
Create LIT5833ReadingGospel.xml

### PR DESCRIPTION
**Pull request by DariaElagina**
The text of the incipit will be provided in the manuscript description

### CODE REVIEW

File: new/LIT5833ReadingGospel.xml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT5833ReadingGospel" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">On the Reading of Gospel</title>
```
Comment (thea-m) [inline new/LIT5833ReadingGospel.xml]: en

File: new/LIT5833ReadingGospel.xml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="ar">Arabic</language>
+                <language ident="it">Italian</language>
+                <language ident="am">Amharic</language>
```
Comment (thea-m) [inline new/LIT5833ReadingGospel.xml]: I think it's a good practice to delete the unused languages, but of course not necessary

File: new/LIT5833ReadingGospel.xml (hunk 1 above)
Comment (DariaElagina) [reply new/LIT5833ReadingGospel.xml]: sure, thanks

File: new/LIT5833ReadingGospel.xml [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
-schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
 type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 <TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT5833ReadingGospel" xml:lang="en" type="work">
     <teiHeader>
         <fileDesc>
             <titleStmt>
-                <title xml:lang="gez" xml:id="t1">On the Reading of Gospel</title>
+                <title xml:lang="en" xml:id="t1">On the Reading of Gospels</title>
```
Comment (thea-m) [inline new/LIT5833ReadingGospel.xml]: actually, the Gospels, I believe. Sorry!

Comment (DariaElagina) [reply new/LIT5833ReadingGospel.xml]: :)

**Comment by DariaElagina** [comment]
As far as I could check, using Zuurmond, it is not. And the text appears at the end of the Gospels

**Pull request merged**